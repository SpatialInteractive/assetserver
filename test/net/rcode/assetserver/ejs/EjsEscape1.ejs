This file has top level directive escapes.  The following should
escape properly
###EJSON
Some intervening
###EJSDISABLE
