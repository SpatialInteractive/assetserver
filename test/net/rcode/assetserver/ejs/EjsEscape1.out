{Literal(1,0,121)='This file has top level directive escapes.  The following should\nescape properly\n##EJSON\nSome intervening\n##EJSDISABLE\n'}
